nidm/core/tests/test_provone.py

```diff
@@ -0,0 +1,127 @@
+from nidm.core.provone import ProvONEDocument
+from nidm.core import Constants
+from nidm.core.dot import provone_to_dot
+import pytest
+
+
+@pytest.fixture(scope="module")
+def doc():
+    # Create new provone document with namespaces
+    d1 = ProvONEDocument()
+    d1.add_namespace('dcterms', 'http://purl.org/dc/terms/')
+    d1.add_namespace('wfms', 'http://www.wfms.org/registry/')
+    d1.add_namespace('rdfs', 'http://www.w3.org/2000/01/rdf-schema#')
+    d1.add_namespace('nowpeople', 'http://www.provbook.org/nownews/people/')
+    d1.add_namespace('xsd', 'http://www.w3.org/2001/XMLSchema#')
+    d1.add_namespace('owl', 'http://www.w3.org/2002/07/owl#')
+
+    return d1
+
+
+def test_ispartof(doc):
```

Comment:
these tests should not just test writing out a file, it should also compare or determine what was written to be correct. perhaps @djarecka can help @sanuann to improve these tests.